samples/subsys/display/lvgl/nxp_mipi.conf

```diff
@@ -0,0 +1,6 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_DMA=y
+CONFIG_MAIN_STACK_SIZE=20480
+CONFIG_MIPI_DSI=y
```

Comment:
What is the rationale to not have this as a board overlay conf instead?

Reply:
> What is the rationale to not have this as a board overlay conf instead?

@faxe1008 I do not want this to be added to each platform, as this is generally applies to all NXP platform. so only one config is added.